src/components/CurrentDirectors/CurrentDirectors.vue

```diff
@@ -0,0 +1,157 @@
+              <v-tooltip top :disabled="formatFullName(orgPerson.officer).length < 25" color="primary">
+                <template v-slot:activator="{ on }">
+                  <span v-on="on" class="director-name">{{ formatFullName(orgPerson.officer).toLowerCase() }}</span>
+                </template>
+                <span class="director-name">{{ formatFullName(orgPerson.officer).toLowerCase() }}</span>
+              </v-tooltip>
+            </v-col>
+
+            <!-- Mailing Address -->
+            <v-col class="pr-5">
+              <base-address class="director-detail" :address="orgPerson.mailingAddress" />
+            </v-col>
+
+            <!-- Delivery Address -->
+            <v-col class="pr-5">
+              <p v-if="isSame(orgPerson.mailingAddress, orgPerson.deliveryAddress)"
+                class="director-detail">Same as Mailing Address
+              </p>
+              <base-address v-else class="director-detail" :address="orgPerson.deliveryAddress"/>
+            </v-col>
+
+            <!-- Appoinment Date -->
+            <v-col class="pr-5">
+              <span class="director-detail">{{ orgPerson.roles[0].appointmentDate }} to Current</span>
+            </v-col>
+          </v-row>
+        </div>
+      </div>
+    </v-card>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Mixins } from 'vue-property-decorator'
+import { Getter } from 'vuex-class'
+import { OrgPersonIF } from '@/interfaces'
+import { RoleTypes } from '@/enums'
+import { CommonMixin } from '@/mixins'
+import BaseAddress from 'sbc-common-components/src/components/BaseAddress.vue'
+
+@Component({
+  components: {
+    BaseAddress
+  }
+})
+export default class CurrentDirectors extends Mixins(CommonMixin) {
+  // Declarations for template
+  readonly RoleTypes = RoleTypes
+
+  // Global getters
+  @Getter getPeopleAndRoles!: OrgPersonIF[]
+
+  /** Headers for the person table. */
+  readonly tableHeaders = ['Name', 'Mailing Address', 'Delivery Address', 'Effective Dates']
+
+  /**
+   * Called when component is mounted.
+   */
+  private mounted (): void {}
+
+  /**
+   * Returns the current directors from the people and roles.
+   * @returns array of directors
+   */
+  private get currentDirectors () : OrgPersonIF[] {
+    const directors = this.getPeopleAndRoles
+      .filter(people => people.roles.some(role =>
+        role.roleType.toLowerCase() === 'director' && role.cessationDate === null))
+    for (const director of directors) {
+      director.roles = director.roles.filter(role => role.roleType.toLowerCase() === 'director')
+    }
+    return directors
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import '@/assets/styles/theme.scss';
+
+.header-container {
+  display: flex;
+  background-color: $BCgovBlue5O;
+  padding: 1.25rem;
+}
+
+[class^="col"] {
+  padding-top: 0;
+  padding-bottom: 0;
+}
+
+.instructional-text {
+  font-size: 16px;
```

Comment:
Except for fine-tuning spacing (eg, a few px here or there), we specify sizes in rems.

https://www.google.com/search?q=why+use+rem+instead+of+px

PS 16px = 1 rem

PPS You should try to use an existing style for this instructional text. See:
- [the Vuetify built in styles](https://vuetifyjs.com/en/styles/text-and-typography/#typography)
- https://github.com/bcgov/business-edit-ui/blob/master/src/assets/styles/base.scss
- https://github.com/bcgov/business-edit-ui/blob/master/src/assets/styles/overrides.scss

Reply:
This comment applies to the other font sizes in this file, and to the other classes you're using (better to not have one-time classes here esp. when existing classes probably provide the styles you need).

Reply:
Ok the design had all it's sizes in px, I'll take a look at those style though

Reply:
I know :( I have asked them to specify rems many times but ...

Anyway, it's easy for us to convert px -> rem.